tests/components/PixLink.test.js

```diff
@@ -0,0 +1,246 @@
+        const urls = [
+          'https://pro.pix.fr/',
+          'https://pro.pix.fr/une-page-de-pro',
+          'https://pix.org/fr',
+          'https://pix.org/fr/une-page-si-t-es-pas-en-france',
+          'https://pix.org/fr-be',
+          'https://pix.org/fr-be/une-fois',
+          'https://pix.org/en-gb',
+          'https://pix.org/en-gb/tea-time',
+          'https://google.com/',
+        ]
+
+        for (const url of urls) {
+          expect(removeHostIfCurrentSite(url, locale)).toBe(url)
+        }
+      })
+    })
+
+    describe('when current site is pix.org', () => {
+      beforeEach(() => {
+        process.env.SITE = SITES_PRISMIC_TAGS.PIX_SITE
+        locale = 'fr'
+      })
+
+      it('should remove host from URLs under https://pix.org', () => {
+        const urls = [
+          ['https://pix.org/', '/'],
+          ['https://pix.org/fr', '/fr'],
+          ['https://pix.org/fr/pourquoi', '/fr/pourquoi'],
+          [
+            'https://pix.org/fr/la/totale?foo=bar#la-precision',
+            '/fr/la/totale?foo=bar#la-precision',
+          ],
+          ['https://pix.org/fr-be/une-fois', '/fr-be/une-fois'],
+          ['https://pix.org/en-gb/tea-time', '/en-gb/tea-time'],
+        ]
+
+        for (const [url, expected] of urls) {
+          expect(removeHostIfCurrentSite(url, locale)).toBe(expected)
+        }
+      })
+
+      it('should not remove host from URLs outside of https://pix.org', () => {
+        const urls = [
+          'https://pro.pix.fr/',
+          'https://pro.pix.fr/une-page-de-pro',
+          'https://pix.fr',
+          'https://pix.fr/une-page-de-la-france',
+          'https://google.com/',
+        ]
+
+        for (const url of urls) {
+          expect(removeHostIfCurrentSite(url, locale)).toBe(url)
+        }
+      })
+    })
+
+    describe('when current site is pro.pix.fr', () => {
+      beforeEach(() => {
+        process.env.SITE = SITES_PRISMIC_TAGS.PIX_PRO
+      })
+
+      it('should remove host from URLs under https://pro.pix.org', () => {
+        const urls = [
+          ['https://pro.pix.fr/', '/'],
+          ['https://pro.pix.fr/une-page-de-pro', '/une-page-de-pro'],
+        ]
+
+        for (const [url, expected] of urls) {
+          expect(removeHostIfCurrentSite(url, locale)).toBe(expected)
+        }
+      })
+
+      it('should not remove host from URLs outside of https://pro.pix.org', () => {
+        const urls = [
+          'https://pix.fr',
+          'https://pix.fr/une-page-de-la-france',
+          'https://pix.org/',
+          'https://pix.org/fr',
+          'https://pix.org/fr/pourquoi',
+          'https://pix.org/fr-be/une-fois',
+          'https://pix.org/en-gb/tea-time',
+          'https://google.com/',
+        ]
+
+        for (const url of urls) {
+          expect(removeHostIfCurrentSite(url, locale)).toBe(url)
+        }
+      })
+    })
+  })
+
+  describe('getPathLocale()', () => {
```

Comment:
```suggestion
  describe('#getPathLocale', () => {
```
Pour matcher avec le nommage utilisé chez Pix.